addons/mail/models/mail_thread.py

```diff
@@ -1771,6 +1771,9 @@ def _mail_find_partner_from_emails(self, emails, records=None, force_create=Fals
         done_partners += [partner for partner in partners]
         remaining = [email for email in normalized_emails if email not in [partner.email_normalized for partner in done_partners]]
 
+        # prioritize current user if exists in list
```

Comment:
For commit message: maybe "[FIX] mail: fix current user finding in _mail_find_partner_from_emails" ?

Reply:
I changed it, thanks for feedback :pray: